test/integration/foreman_puppet/smartclass_parameter_js_test.rb

```diff
@@ -74,6 +74,7 @@ class SmartclassParameterJSTest < IntegrationTestWithJavascript
       within(:xpath, '//table') do
         click_link 'ssl'
       end
+      page.driver.browser.file_detector = nil if page.driver.browser.respond_to?(:file_detector=)
```

Comment:
If this is a workaround for buggy behavior, I would suggest adding a comment with a link to the Redmine or a bug report.
Also, shouldn't this be a part of before/after test setup actions?

Reply:
AFAIK this is a problem with the selenium ruby bindings https://stackoverflow.com/questions/70441796/selenium-webdriver-for-aws-device-farm-error-when-sending-period-keystroke-t

another more implicit fix is described here https://github.com/theforeman/foreman_puppet/issues/382 to just rename `test` to something that is not locally available